Sources/CustomerCenter/CustomerCenterConfigData.swift

```diff
@@ -452,17 +452,33 @@ public struct CustomerCenterConfigData: Equatable {
             public let title: String
             public let subtitle: String
             public let productMapping: [String: String]
+            public let crossProductPromotions: [String: CrossProductPromotion]
+
+            public struct CrossProductPromotion: Equatable {
+                public let storeOfferIdentifier: String
+                public let targetProductId: String
+
+                public init(
+                    storeofferingidentifier: String,
+                    targetproductid: String
+                ) {
+                    self.storeOfferIdentifier = storeofferingidentifier
+                    self.targetProductId = targetproductid
+                }
+            }
```

Comment:
nit: I think we should stop nesting types. It's absolutely annoying for a type that's internal. Ideally we should flatten all the types, and make them `spi`.

We can do that later, of course. End of rant

Reply:
yeah honestly there's a `nesting` linter rule for that reason that we are ignoring. I completely agree